spec/search-api.yaml

```diff
@@ -108,6 +108,12 @@ components:
             $ref: "#/components/schemas/Table"
         pagination:
           $ref: '#/components/schemas/Pagination'
+        errors:
+          type: array
+          description: List of errors encountered
+          items:
+            $ref: "#/components/schemas/Error"
+      additionalProperties: false
```

Comment:
@patmagee I think this is a rebase error - see adbcb06fb08e77ecd7899209694b10831df25ee2. Can you confirm?